.github/workflows/test-and-build.yml

```diff
@@ -25,14 +25,28 @@ jobs:
     with:
       python_version: "3.10"
 
-  call-docker-ghcr-workflow:
+  call-docker-ghcr-workflow-cpu:
     needs: call-version-info-workflow
     # Docs: https://github.com/ASFHyP3/actions
-    uses: ASFHyP3/actions/.github/workflows/reusable-docker-ghcr.yml@v0.11.2
+    uses: ASFHyP3/actions/.github/workflows/reusable-docker-ghcr.yml@specify-docker-file
     with:
-      version_tag: ${{ needs.call-version-info-workflow.outputs.version_tag }}
+      version_tag: ${{ needs.call-version-info-workflow.outputs.version_tag }}.cpu
       release_branch: main
       develop_branch: develop
       user: tools-bot
+      file: Dockerfile.cpu
+    secrets:
+      USER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  call-docker-ghcr-workflow-gpu:
+    needs: call-version-info-workflow
+    # Docs: https://github.com/ASFHyP3/actions
+    uses: ASFHyP3/actions/.github/workflows/reusable-docker-ghcr.yml@specify-docker-file
+    with:
+      version_tag: ${{ needs.call-version-info-workflow.outputs.version_tag }}.gpu
+      release_branch: main
+      develop_branch: develop
+      user: tools-bot
+      file: Dockerfile
```

Comment:
The README refers to `Dockerfile.gpu`, so probably simplest to revert back to `Dockerfile` for CPU and `Dockerfile.gpu` for GPU so we don't have to update the README.